test-packages/framework/projects/cap/app/manage_travels/webapp/localService/metadata.xml

```diff
@@ -1,1820 +1,1392 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Do we need this file for a CDS project?

Reply:
for cds project no, but needed for ui5 project. [Here](https://github.com/SAP/ui5-language-assistant/pull/523/files#diff-47b1cdd6a07cb38e18cc1ae6229e9bb5026db16f4cd01e8a5071676f9fb2a76fR48) use it for ui5 project.